twilight-lavalink/src/http.rs

```diff
@@ -258,7 +249,32 @@ pub fn load_track(
 ) -> Result<Request<&'static [u8]>, HttpError> {
     let identifier =
         percent_encoding::percent_encode(identifier.as_ref().as_bytes(), NON_ALPHANUMERIC);
-    let url = format!("http://{address}/loadtracks?identifier={identifier}");
+    let url = format!("http://{address}/v4/loadtracks?identifier={identifier}");
+
+    let mut req = Request::get(url);
+
+    let auth_value = HeaderValue::from_str(authorization.as_ref())?;
+    req = req.header(AUTHORIZATION, auth_value);
+
+    req.body(b"")
+}
+
+/// Decode a single track into its info
```

Comment:
```suggestion
/// Decode a single track into its info.
```

I also think info sounds a bit weird, `information` at the very least should be better.

Reply:
Done. I updated the phrasing. I did just get the original from the actual docs: https://lavalink.dev/api/rest.html#track-decoding